README.md

```diff
@@ -1,45 +1,8 @@
-# Ruby on Rails Learning Path
```

Comment:
Please do not make changes on the readme file.

Reply:
Sorry, miss clicked